### PR TITLE
Add alias for AppNexusDirect and generate new build

### PR DESCRIPTION
**Pull request by katebee**
Required for including AppNexus directly

PR to frontend to follow 😸 😸

### CODE REVIEW

File: modules/appnexusBidAdapter.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const SOURCE = 'pbjs';
 
 export const spec = {
   code: BIDDER_CODE,
-  aliases: ['appnexusAst', 'brealtime', 'emxdigital', 'pagescience', 'defymedia', 'gourmetads', 'matomy', 'featureforward', 'oftmedia', 'districtm', 'xhb'],
+  aliases: ['appnexusAst', 'and', 'brealtime', 'emxdigital', 'pagescience', 'defymedia', 'gourmetads', 'matomy', 'featureforward', 'oftmedia', 'districtm', 'xhb'],
```
Comment (jeteve) [inline modules/appnexusBidAdapter.js]: 'and' ?

Comment (jeteve) [reply modules/appnexusBidAdapter.js]: Ha! 'App Nexus Direct' :) My little brain got confused

**Comment by jeteve** [comment]
I think you also need to include this adapter in the build now that we need it clientside? or is it already in?

**Comment by janua** [comment]
:+1:

**Comment by katebee** [comment]
> I think you also need to include this adapter in the build now that we need it clientside? or is it already in?

I have a `TODO` left on the frontend PR to add this in. Just needed to wait until this one is merged 😁

**Pull request merged**